Exiled.Events/Patches/Generic/PlayerCurrentBroadcast.cs

```diff
@@ -0,0 +1,117 @@
+// -----------------------------------------------------------------------
+// <copyright file="PlayerCurrentBroadcast.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Generic
+{
+#pragma warning disable SA1313
+    using System.Collections.Generic;
+    using System.Linq;
+    using Exiled.API.Features;
+    using HarmonyLib;
+
+    using Hints;
+
+    using MEC;
+    using Mirror;
+
+    using BaseBroadcast = Broadcast;
+
+    /// <summary>
+    /// Patches <see cref=""/>.
```

Comment:
No. Fix this summary.

Reply:
Also I would highly encourage using a simple transpiler for the patches here, simply for patch overhead efficiency.